lib/web_ui/lib/src/engine/text/canvas_paragraph.dart

```diff
@@ -5,9 +5,9 @@
 import 'package:ui/ui.dart' as ui;
 
 import '../dom.dart';
-import '../embedder.dart';
 import '../html/bitmap_canvas.dart';
 import '../profiler.dart';
+import '../view_embedder/dom_manager.dart';
```

Comment:
(Yes, it's odd to `import dom_manager.dart` to end up using a class named `StyleManager` :P)

Reply:
Yep!